### PR TITLE
Feat/tr 2757/slider interaction rtl

### PR DESCRIPTION
**Pull request by shaveko**
related to https://oat-sa.atlassian.net/browse/TR-2757

### Description 

Detect the interation is in RTL mode and forward this info to the noUI-slider plugin, invert the labels

### How to test

Should be tested with changes in tao base package (PR is here : https://github.com/oat-sa/tao-core/pull/3234)

### CODE REVIEW

**Review by marpesia — CHANGES_REQUESTED** [review]
It works well, good job.
But I have to delete the package-lock.json to test it. Otherwise, I can run `npm i`.
Can you check why yours is different?
---
As this is a package extension, a PR on the `extension-tao-itemqti` will be needed.
You need to link the extension to the package in the `views/package.json` like that
`"@oat-sa/tao-item-runner-qti": "github:oat-sa/tao-item-runner-qti-fe#feat/TR-2757/slider-interaction-rtl"`
And then bundle the files. Ask me please if you have any doubt

**Comment by meefox** [comment]
Adding, for faster review it would be good to have a complete composer file that will have reference to all custom branches needed for this ticket. 
That makes it easier to see the whole picture of changes, and it is simpler will be to deploy to QA later on


Master composer for RTL, you can use this https://github.com/oat-sa/kitchen-playground/blob/mca/test/files/delivery/composer.json
with modifications:
`"oat-sa/tao-core": "dev-fix/tr-2757/slider-interaction-rtl as 48.41.4",`
`"oat-sa/extension-tao-itemqti": "dev-fix/tr-2757/slider-interaction-rtl as 48.41.4 as 28.17.0",` <- this need to be created with changes that Maria mentioned to get changes in this branch.

File: src/qtiCommonRenderer/renderers/interactions/SliderInteraction.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ var _slideTo = function(options) {
 const render = function(interaction) {
     const attributes = interaction.getAttributes(),
         $container = interaction.getContainer(),
+        direction = window.getComputedStyle($container[0]).getPropertyValue('direction') || 'ltr',
```
Comment (meefox) [inline src/qtiCommonRenderer/renderers/interactions/SliderInteraction.js:47]: **suggestion for the future** 
@jsconan / @krampstudio 
here is the same issue that I had with Liquid PCI, changing language in Authoring will not reflect on the component. 
Authoring is not in the scope of this task but in the future it worth having generic utils/helpers to handle change and rerender the component.

Comment (krampstudio) [reply src/qtiCommonRenderer/renderers/interactions/SliderInteraction.js:47]: The approach used by now was for the authoring to render the item. Isn't it the case already? From a runtime perspective, it's acceptable.

Comment (shaveko) [reply src/qtiCommonRenderer/renderers/interactions/SliderInteraction.js:47]: > **suggestion for the future** @jsconan / @krampstudio here is the same issue that I had with Liquid PCI, changing language in Authoring will not reflect on the component. Authoring is not in the scope of this task but in the future it worth having generic utils/helpers to handle change and rerender the component.

Do you mean the item component here? Direction is applied on the item component as well

**Comment by shaveko** [comment]
> It works well, good job!
> 
> But I have to delete the package-lock.json to test it. Otherwise, I can run `npm i`. Can you check why yours is different?
> 
> As this is a package extension, a PR on the `extension-tao-itemqti` will be needed. You need to link the extension to the package in the `views/package.json` like that `"@oat-sa/tao-item-runner-qti": "github:oat-sa/tao-item-runner-qti-fe#feat/TR-2757/slider-interaction-rtl"` And then bundle the files. Ask me please if you have any doubt

Maria, with the help of @meefox we created test enviromnemt
Plz check here:
https://github.com/oat-sa/extension-tao-itemqti/pull/1954

**Review by meefox — APPROVED** [review]

**Review by meefox — APPROVED** [review]
- [ ] New code is covered by tests
- [ ] Tests are running successfully (old and new ones) on my local machine
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by marpesia — APPROVED** [review]
Good job!!!

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Commits messages are meaningful
- [ ] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**